content/ru/_solutions/itvision_office.md

```
@@ -0,0 +1,115 @@
+
+Все эти меры существенно упростили работу инженерного отдела. Если раньше персонал мог обнаружить протечки только по факту значительного подтопления, теперь система сразу отправляет сигнал об инциденте. Мониторинг потребления электроэнергии показывает текущие показатели в реальном времени, позволяет быстро реагировать на нештатные ситуации.
+
+Контроллер Wiren Board доказал свою надёжность, масштабируемость и удобство настройки. Интегратор отметил, что система работает стабильно и обеспечивает 100% аптайм. В результате заказчик остался доволен гибкостью решения и прозрачностью процесса. В планах — интегрировать вентустановки и расширить функционал, чтобы еще точнее контролировать различные параметры и свести к минимуму человеческий фактор.
+
+В ассортименте компании Wiren Board есть устройства для решения как простых, так и сложных задач автоматизации и диспетчеризации: программируемые логические контроллеры, датчики климата, счетчики электроэнергии, сетевые карты для холодильников, мощные релейные модули и т.д. Все это позволяет подстроиться под любые требования заказчика и использовать оборудование Wiren Board в его экосистеме. При этом обеспечивается возможность расширения экосистемы в будущем.
+
+Компания Wiren Board даёт на всё оборудование двухлетнюю гарантию с заменой оборудования без предварительной отправки в сервисный центр, а инженеры технической поддержки готовы помочь в решении как типовых, так и нестандартных задач.
+
+## Дополнительные материалы
+
+- [Мониторинг vs расчеты: где взять недостающую электрическую мощность и попутно решить проблемы](https://habr.com/ru/companies/wirenboard/articles/869138/)
```

Markdownlint (0.37.0)

Expected: asterisk; Actual: dash
Unordered list style

(MD004, ul-style)